components/brave_wallet_ui/common/slices/api.slice.ts

```diff
@@ -1716,11 +1717,60 @@ export function createWalletApi () {
                 chainId: null
               })
       }),
+      sendBtcTransaction: mutation<
+        { success: boolean },
+        SendBtcTransactionParams
+      >({
+        queryFn: async (payload, { dispatch }, extraOptions, baseQuery) => {
+          try {
+            const { txService } = baseQuery(undefined).data
+
+            const btcTxData: BraveWallet.BtcTxData = {
+              to: payload.to,
+              amount: BigInt(payload.value),
+              fee: BigInt(0),
```

Comment:
Why is this field `0`? Also, shouldn't this field be called `feePerByte`?

Reply:
Not used by backend.